python/xorq/backends/let/tests/test_api.py

```diff
@@ -51,7 +51,14 @@ def test_executed_on_original_backend(parquet_dir, csv_dir, mocker):
 
 
 def test_read_postgres():
-    uri = "postgres://postgres:postgres@localhost:5432/ibis_testing"
+    con = xo.postgres.connect_env()
+    uri = (
+        f"postgres://{con.con.info.user}:"
+        f"{con.con.info.password}@"
+        f"{con.con.info.host}:"
+        f"{con.con.info.port}/"
+        f"{con.con.info.dbname}"
+    )
```

Comment:
I think in here we can do: 
```suggestion
import os
uri = (
    f"postgres://{os.environ['POSTGRES_USER']}:"
    f"{os.environ['POSTGRES_PASSWORD']}@"
    f"{os.environ['POSTGRES_HOST']}:"
    f"{os.environ['POSTGRES_PORT']}/"
    f"{os.environ['POSTGRES_DATABASE']}"
)
```

So we don't need to create a con object